style.css

```diff
@@ -0,0 +1,46 @@
+html, body {
+  height: 100%;
+}
+
+.table-container {
+  overflow-x: auto;
+  overflow-y: auto;
+}
+
+.table-container__table {
+  border-collapse: collapse;
+  font-size: 18pt;
+  width: 100%;
+  height: 150px;
+  position: absolute;
+  top: 35%;
+}
+
+.table-container__table tr:nth-child(even) {
+  background-color: dimgray;
+}
+
+.table-container__table tr:nth-child(odd) {
+  background-color: darkgray;
+}
+
+.table-container__table td, th {
+  border: 1px solid black;
+  min-width: 80px;
+}
+
+.table-container__table th {
+  text-align: left;
+  border-bottom: 1px solid black;
+  border-right: 1px solid black;
+  background-color: lightgray;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.table-container__table th:hover {
+  cursor: pointer;
+}
+
+.table-container__table td {
+  border-right: 1px solid black;
+}
```

Comment:
you can you use classes you gave your elements for this or create a new class (one for header and one for cells) and style them